pinot-common/src/test/java/com/linkedin/pinot/common/segment/fetcher/testSegmentFetcher.java

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright (C) 2014-2016 LinkedIn Corp. (pinot-core@linkedin.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.linkedin.pinot.common.segment.fetcher;
+
+import java.io.File;
+import java.util.Map;
+
+/**
+ * Created by jamesshao on 9/20/17.
+ */
+public class testSegmentFetcher implements SegmentFetcher {
```

Comment:
Is this a test class or a DummySegmentFetcher?

Reply:
it is supposed to be a dummy fetcher that we used for unit test (see pinot-common/src/test/java/com/linkedin/pinot/common/segment/fetcher/SegmentFetcherFactoryTest.java)